src/java.base/share/classes/jdk/internal/foreign/SystemLookup.java

```diff
@@ -126,16 +125,17 @@ private static SymbolLookup libLookup(Function<RawNativeLibraries, NativeLibrary
         };
     }
 
-    /*
-     * Returns the path of the given library name from JDK
-     */
-    private static Path jdkLibraryPath(String name) {
-        Path javahome = Path.of(StaticProperty.javaHome());
-        String lib = OperatingSystem.isWindows() ? "bin" : "lib";
-        String libname = System.mapLibraryName(name);
-        return javahome.resolve(lib).resolve(libname);
+    private static SymbolLookup stdLibLookup(Function<RawNativeLibraries, NativeLibrary> loader) {
+        NativeLibrary lib = loader.apply(RawNativeLibraries.newInstance(MethodHandles.lookup()));
+        return lookup(lib);
     }
 
+    @SuppressWarnings("restricted")
+    private static SymbolLookup sysLookup() {
+        NativeLibraries libs = jdk.internal.loader.NativeLibraries.newInstance(null);
```

Comment:
```suggestion
        NativeLibraries libs = NativeLibraries.newInstance(null);
```
We already imported this class.